tools/webdriver/webdriver/undefined.py

```diff
@@ -0,0 +1,6 @@
+class Undefined:
```

Comment:
Can't you just write this as:
```
UNDEFINED = object()
```
if you're just trying to create a unique sentinel value and don't actually care about its type?

Reply:
We do care about the type though. We use it in other PRs.

Reply:
For example, `Union[str,Undefined]` is far more informative then `Union[str,object]`.